### ISSUE TITLE
Implement Display instances

### ISSUE THREAD

**Issue by colbyn** [g-plane/raffia#5]
It would be nice and convenient to have `Display` instances implemented for all AST data types for for rendering CSS syntax from a transformed AST.

**Comment by colbyn** [comment]
Especially because rust trait rules forbid instances on out of crate data types. I'd practically have to reimplement the entire AST for this otherwise simple task...

**Comment by colbyn** [comment]
Started working on preliminary support for it [over here](https://github.com/colbyn/raffia) (although I only need display instances for a small subset of the types defined in this library).

**Comment by g-plane** [comment]
Do you mean generating from AST to code?

**Comment by colbyn** [comment]
Yeah, after manipulating the AST I needed a way of render CSS syntax from the associated `Stylesheet` datatype (or whatever it was called). But I actually ended up using `swc_css_codegen`. Anyway this isn’t relevant to me anymore so I suppose I’ll close the issue. Thanks! :)
